backend/engine/plugins/ghas_secrets/validator.py

```diff
@@ -0,0 +1,62 @@
+import subprocess
+from typing import Tuple
+from artemislib.logging import Logger
+
+LOG = Logger("ghas_secrets")
+
+valid_types = {
+    "wiki_commit": True,
+    "issue_title": True,
+    "issue_body": True,
+    "issue_comment": True,
+    "discussion_title": True,
+    "discussion_body": True,
+    "discussion_comment": True,
+    "pull_request_title": True,
+    "pull_request_body": True,
+    "pull_request_comment": True,
+    "pull_request_review": True,
+    "pull_request_review_comment": True,
+}
+
+
+def validate(location: dict, path: str) -> Tuple[bool, str, str]:
```

Comment:
(minor) We can use the built-in `tuple` instead of `typing.Tuple` [since we're on Python 3.9](https://docs.python.org/3/library/typing.html#typing.Tuple).